webpack.common.js

```diff
@@ -1,13 +1,12 @@
-var CleanWebpackPlugin = require('clean-webpack-plugin'),
-    config = require('./config.json'),
-    LodashModuleReplacementPlugin = require('lodash-webpack-plugin'),
+var CleanPlugin = require('clean-webpack-plugin'),
```

Comment:
var?

Reply:
Admittedly, the codebase has not been cleaned up fully to work with ES6 syntax. I can do it for the webpack configuration, but I don't want to do a full code cleanup as part of this PR.